src/classes/UTIL_Describe_TEST.cls

```diff
@@ -101,6 +101,37 @@ public with sharing class UTIL_Describe_TEST {
         try {
             picklistEntries = UTIL_Describe_API.getPicklistActiveValuesMap('Account', 'id');
         } catch(UTIL_Describe.SchemaDescribeException sde){}
-    }       
+    } 
 
+    // Test for childRelationshipNamesService functionality
+    @isTest
+    private static void childRelationshipNamesServiceTest() {
+
+        // Insert Account
+        Account adminAccount = UTIL_UnitTestData_API.getMultipleTestAccounts(1, UTIL_Describe_API.getAdminAccRecTypeID())[0];
+        adminAccount.Name = 'Test Account 000';
+        insert adminAccount;
+
+        Test.startTest();
+            Schema.SObjectType sObjectTypeAcc = adminAccount.getSObjectType();
+            List<String> allChildRelationshipNames = UTIL_Describe.getAllChildRelationshipNames(sObjectTypeAcc);
+        Test.stopTest();
+
+        System.assertNotEquals(NULL, allChildRelationshipNames);
+        System.assertNotEquals(0, allChildRelationshipNames.size());
+        System.assert(allChildRelationshipNames.contains('Contacts'));
+    }
+
+    // NULL Test for childRelationshipNamesService functionality
+    @isTest
+    private static void childRelationshipNamesServiceNegativeTest() {
+        try {
+            Test.startTest();
+                List<String> allChildRelationshipNames = UTIL_Describe.getAllChildRelationshipNames(NULL);
+            Test.stopTest();
+        } catch (Exception Ex) {
+            // Should throw a validation message, catching the message to the test the exception
+            System.assert(ex.getMessage().contains('Attempt to de-reference a null object'));
+        }
```

Comment:
@skosgi1 Please change the structure of this to handle the exception and test for not throwing an exception or encountering an unexpected exception. We can accomplish this by placing a System.assert(false,message) at the end of the try block and have an empty catch block for System.NullPointerException.